### ISSUE TITLE
End of Grooveshark? End of Spotify-To-MP3?

### ISSUE THREAD

**Issue by martinbetz** [frosas/spotify-to-mp3#53]
Grooveshark is out of business. This app heavily relies on Grooveshark. Is there any alternative? Or is this app dead?

**Comment by om23** [comment]
Someone put a copycat of groveshark: http://grooveshark.io/

**Comment by martinbetz** [comment]
You can try to fork this gem substituting the old URL with the new clone. I doubt that it will work. I tested some songs I listened to in the old Grooveshark and could not find them in the new clone.

**Comment by frosas** [comment]
I agree. Both the search engine and the tracks availability seems to be quite poor for now.

You can consider this app pretty much dead now :(

**Comment by cxdy** [comment]
Pretty late, but you can try Spotify-to-VK

https://github.com/nazar-pc/spotify-to-vk

**Comment by SathyaBhat** [comment]
I used to use this project pretty heavily and it was unfortunate that grooveshark shutdown. I've started a project which makes use of YouTube to download.

https://github.com/SathyaBhat/spotify-dl

As of now you still need to use youtube-dl to download, but I'll be integrating that soon enough
